config/application.rb

```diff
@@ -29,5 +29,16 @@ class Application < Rails::Application
     config.i18n.default_locale = :de
     config.time_zone = 'Vienna'
     config.autoload_paths += %W["#{config.root}/app/services/"]
+
+    unless Rails.env.test?
+      config.after_initialize do
+        Rails.application.reload_routes!
+        require "chatbot/operation"
+        #Thread.abort_on_exception = true
+        Thread.new do
+          ChatbotOperation::Initialize.()
+        end
+      end
+    end
```

Comment:
This creates (or updates) the persistent menu and the get started actions in Messenger